ProcessSlave.php

```diff
@@ -384,18 +391,20 @@ protected function handleRequest(\React\Http\Request $request, HttpResponse $res
             );
             $this->shutdown();
         }
+        return $response;
     }
 
-    protected function prepareEnvironment(\React\Http\Request $request)
+    protected function prepareEnvironment(RequestInterface $request)
     {
         $_SERVER = $this->baseServer;
```

Comment:
Why do we even need to populate the server vars? Isn't this already a verify specific behaviour of a "legacy" bridge?